config/cluster.yaml.template

```diff
@@ -0,0 +1,11 @@
+# base cluster settings
+base:
+    cluster_id: <id>
+    cluster_vm_size: <vm-size>
+    cluster_size: <number>
+    cluster_size_low_pri: <number>
+    cluster_username: <username>
+    cluster_password: <password>
+    cluster_ssh_key: <path-to-ssh-key>
+    custom_script: <path to custom script to run on each node> (optional)
+    wait: <true/false>
```

Comment:
in general, we should be adding more comments to this config file. 
some points:
- why is there both ssh_key and password?
- can i really define both size_low_pri and size? (how do we represent mutually exclusive params?)
- how do i know which vm sizes are available to me? (good place to provide a link to the docs)

Even really obvious things:
- what is cluster_size? (is it number of nodes? number of executors? number of cores?)
- what is low-pri? (lets provide a link out)
- what does 'wait' do?